deploy.sh

```diff
@@ -47,4 +47,7 @@ eval "${KUBECTL} create configmap es-config --from-file=es-config --dry-run -o y
 eval "${KUBECTL} create configmap fluentd-config --from-file=docker/fluentd/td-agent.conf --dry-run -o yaml" | eval "${KUBECTL} apply -f -"
 eval "${KUBECTL} create configmap kibana-config --from-file=kibana.yml --dry-run -o yaml" | eval "${KUBECTL} apply -f -"
 
+## Install RBAC policies
+eval "${KUBECTL} apply -f rbac"
+
```

Comment:
Can you add the same for es5/deploy.sh?

I.e. create a `../rbac` symlink and add `eval "${KUBECTL} apply -f rbac"`

I have plans to merge es5 and es2.x, but it still requires more testing.